Packages/manifest.json

```diff
@@ -9,13 +9,13 @@
     "com.unity.ide.vscode": "1.2.5",
     "com.unity.learn.iet-framework": "2.2.1",
     "com.unity.memoryprofiler": "0.5.0-preview.1",
+    "com.unity.multiplayer.tools": "1.0.0-pre.8",
     "com.unity.netcode.gameobjects": "1.0.0-pre.10",
-    "com.unity.multiplayer.tools": "1.0.0-pre.7",
     "com.unity.postprocessing": "3.2.1",
     "com.unity.render-pipelines.universal": "12.1.6",
-    "com.unity.services.authentication": "1.0.0-pre.4",
+    "com.unity.services.authentication": "2.1.1",
     "com.unity.services.lobby": "1.0.0-pre.6",
-    "com.unity.services.relay": "1.0.2",
+    "com.unity.services.relay": "1.0.3",
     "com.unity.test-framework": "1.1.31",
     "com.unity.textmeshpro": "3.0.6",
     "com.unity.timeline": "1.6.4",
```

Comment:
Those should also be updated in the Utilities package's dependencies

Reply:
Although the Utilities package doesn't use Authentication nor Lobby, so I could remove them from the dependencies instead